.github/workflows/pr.yaml

```diff
@@ -75,6 +76,13 @@ jobs:
     with:
       build_type: pull-request
       script: "ci/test_python_singlegpu.sh"
+  conda-python-tests-integration:
```

Comment:
I was expecting to see something here/in this section of the yaml that marks it as optional. Is it just not done yet or taken care of somewhere else?

Reply:
I tried to add it as optional in a few commits like https://github.com/rapidsai/cuml/pull/5881/commits/4cabd821bc584b55a37cf46ac0e250afbcf18c6b but seems like I'm doing it wrong, so been asking how to do it and will update the PR accordingly

Reply:
It turns out I also don't know/can't quickly find how to do this! I thought it would be easy but apparently not :-/

Reply:
Ended up needing the help from AJ and now in the last commit https://github.com/rapidsai/cuml/pull/5881/commits/d6979c5b060d27f725ddca7b138682173235b5d0 is optional now